compat/cpu.h

```diff
@@ -36,23 +36,27 @@ inline int sched_getaffinity(int, size_t, cpu_set_t *) { return ENOTSUP; }
 #endif
 
 #if !defined(CPU_SET)
-#define CPU_SET(cpu, set) (void)0
+inline void CPU_SET(int, cpu_set_t *) {}
 #endif
 
 #if !defined(CPU_CLR)
 #define CPU_CLR(cpu, set) (void)0
 #endif
 
 #if !defined(CPU_ISSET)
-#define CPU_ISSET(cpu, set) false
+inline bool
+CPU_ISSET(int, cpu_set_t *)
+{
+    return false;
+}
 #endif
 
 // glibc prior to 2.6 lacks CPU_COUNT
 #ifndef CPU_COUNT
 #define CPU_COUNT(set) CpuCount(set)
 /// CPU_COUNT replacement
 inline int
-CpuCount(const cpu_set_t *set)
+CpuCount(cpu_set_t *set)
```

Comment:
This is a different case: CpuCount() is not a standard macro. This function is our own invention. It should (continue to) correctly declare that the supplied set will not be modified, using `const`. Doing so will not clash with CPU_COUNT() because CPU_COUNT() is a macro.
```suggestion
CpuCount(const cpu_set_t *set)
```